src/Microsoft.Graph.Core/Models/ODataMethodBooleanResponse.cs

```diff
@@ -0,0 +1,33 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph
+{
+    using Newtonsoft.Json;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Represents an intermediate object used for deserializing OData method responses
+    /// that return a single boolean OData primitive. This type is consumed by code files
+    /// generated with:
+    /// https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/dev/Templates/CSharp/Requests/MethodRequest.cs.tt
+    /// The value of a return type is an object:
+    /// http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.html#sec_ReturnType
+    /// </summary>
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
+    public class ODataMethodBooleanResponse
+    {
+        /// <summary>
+        /// Nullable in case the value property is not present.
+        /// /// </summary>
+		[JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "value", Required = Newtonsoft.Json.Required.Default)]
+        public bool? Value { get; set; }
```

Comment:
nit: mixed tabs and spaces cause alignment issues in Github. I recommend this extension: https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs

Reply:
Done